File: check_online.py
import urllib.request

url = "https://garv1629.github.io/aibuild/"
try:
    req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        print("Length:", len(html))
        print("HTML Content:")
        print(html)
except Exception as e:
    print("Error:", e)

File: check_online.js
import https from 'https';

https.get('https://garv1629.github.io/aibuild/', { headers: { 'User-Agent': 'Mozilla/5.0' } }, (res) => {
  let data = '';
  res.on('data', chunk => data += chunk);
  res.on('end', () => {
    console.log('STATUS:', res.statusCode);
    console.log('DATA:', data);
  });
}).on('error', (err) => {
  console.error('ERROR:', err);
});
